server/src/log/log.service.ts

```diff
@@ -12,13 +12,13 @@ export class LogService {
   ) {}
 
   public async get(
-    username: string | null = null,
+    nickname: string | null = null,
     ip: string | null = null,
     page: number = 0
   ) {
     const docPerPage = 25;
     const query = {
-      ...(username ? { username: { $regex: username } } : {}),
+      ...(nickname ? { nickname: { $regex: nickname } } : {}),
```

Comment:
Request stil use username, but query use nickname. Which one do you want to use?